apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -143,6 +144,7 @@ describe('MultiPayment', () => {
 
                     const integrationClient = await makeClientWithNewRegisteredAndLoggedInUser()
                     await createTestAcquiringIntegrationAccessRight(admin, acquiringIntegration, integrationClient.user)
+                    // TODO(toplenboren): Fix this test
                     const [_, updatedMultiPaymentAttrs] = await updateTestMultiPayment(integrationClient, multiPayment.id, {
```

Comment:
There is no need to write _
```
const [, a] = [1,2]
console.log(a)
// 2
```

Reply:
Will be fixed here: https://github.com/open-condo-software/condo/pull/766